docs/source/user_guide/installation/GNU-linux.rst

```diff
@@ -141,9 +141,9 @@ GNU/Linux and macOS
 
 .. code:: bash
 
-	  pybamm_install_jax
+	  pip install "pybamm[jax]"
 
-The ``pybamm_install_jax`` command is installed with PyBaMM. It automatically downloads and installs jax and jaxlib on your system.
+The ``pip install "pybamm[jax]"`` command is installed with PyBaMM. It automatically downloads and installs jax and jaxlib on your system.
```

Comment:
You might have to rewrite this 🙂 

and add a note here about `pybamm_install_jax` being deprecated.